rust-code-analysis-book/src/commands/nodes.md

```diff
@@ -1,52 +1,45 @@
 # Nodes
 
-**rust-code-analysis-cli** allows to extract some information from the nodes
-which compose the *Abstract Syntax Tree (AST)* of a source code.
+The rust-code-analysis-cli provides commands to analyze and extract information from the nodes in the **AST** of a source file.
 
-## Find Errors
+## Error Detection
 
-To know if there are some syntactic errors in your code, run:
+To detect syntactic errors in your code, run:
 
-```console
-rust-code-analysis-cli -p /path/to/your/file/or/directory -I "*.ext" -f -error
+```bash
+rust-code-analysis-cli -p /path/to/file/or/directory -I "*.ext" -f error
```

Comment:
Let's try to reduce the diff in cases like these.
For example, no need to switch from "console" to "bash", and no need to replace "/path/to/your/file/or/directory" with "/path/to/file/or/directory"